src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -358,20 +362,32 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
             "ui.panel.config.automation.picker.bulk_actions.no_category"
           )}
         </div>
+      </ha-menu-item>
+      <md-divider role="separator" tabindex="-1"></md-divider>
+      <ha-menu-item @click=${this._createCategory}>
+        <div slot="headline">
+          ${this.hass.localize("ui.panel.config.category.editor.add")}
+        </div>
       </ha-menu-item>`;
     const labelItems = html` ${this._labels?.map((label) => {
       const color = label.color ? computeCssColor(label.color) : undefined;
       return html`<ha-menu-item
-        .value=${label.label_id}
-        @click=${this._handleBulkLabel}
-      >
-        <ha-label style=${color ? `--color: ${color}` : ""}>
-          ${label.icon
-            ? html`<ha-icon slot="icon" .icon=${label.icon}></ha-icon>`
-            : nothing}
-          ${label.name}
-        </ha-label>
-      </ha-menu-item>`;
+          .value=${label.label_id}
+          @click=${this._handleBulkLabel}
+        >
+          <ha-label style=${color ? `--color: ${color}` : ""}>
+            ${label.icon
+              ? html`<ha-icon slot="icon" .icon=${label.icon}></ha-icon>`
+              : nothing}
+            ${label.name}
+          </ha-label>
+        </ha-menu-item>
+        <md-divider role="separator" tabindex="-1"></md-divider>
+        <ha-menu-item @click=${this._createLabel}>
+          <div slot="headline">
+            ${this.hass.localize("ui.panel.config.labels.add_label")}
+          </div>
+        </ha-menu-item>`;
```

Comment:
It should move outside the map 😅

![CleanShot 2024-04-03 at 13 06 03](https://github.com/home-assistant/frontend/assets/5878303/7d65d14c-4a54-4a24-a512-2d8858d3ff99)

Reply:
You think? 😄